app/src/main/java/org/mozilla/rocket/content/travel/ui/adapter/CityCategoryAdapterDelegate.kt

```diff
@@ -35,7 +35,14 @@ class CityCategoryViewHolder(
 
     override fun bind(uiModel: DelegateAdapter.UiModel) {
         val cityCategory = uiModel as CityCategoryUiModel
-        category_title.text = cityCategory.subcategoryName
+
+        category_title.text =
+            when (cityCategory.subcategoryName) {
```

Comment:
Please follow what we did in shopping and game verticals that use subcategory id to replace the localized category name.
https://github.com/mozilla-tw/FirefoxLite/blob/master/app/src/main/java/org/mozilla/rocket/content/ecommerce/ui/ShoppingMapper.kt#L83